### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/org/zaproxy/zap/extension/tips/resources/Messages_zh_CN.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #X-Generator: crowdin.com
 # Strings used by the add-on
 tips.desc=\u63d0\u793a\u548c\u6280\u5de7
-tips.button.allTips=All tips
-tips.button.nextTip=Next tip
+tips.button.allTips=Alle tips
+tips.button.nextTip=Pr\u00f3ximo Consejo
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/tips/resources/Messages_zh_CN.properties]: This PR has also mixed translations.

Comment (kingthorin) [inline src/org/zaproxy/zap/extension/tips/resources/Messages_zh_CN.properties]: Sorry I missed that. Have you reverted via crowdin or do you need me to do something? I don't mind just point me in type right direction.

Comment (thc202) [inline src/org/zaproxy/zap/extension/tips/resources/Messages_zh_CN.properties]: It was reverted in Crowdin, we can wait for the next @zapbot PR to re-sync?

Comment (kingthorin) [inline src/org/zaproxy/zap/extension/tips/resources/Messages_zh_CN.properties]: Yup, sound fine to me

Comment (thc202) [inline src/org/zaproxy/zap/extension/tips/resources/Messages_zh_CN.properties]: OK.

**Pull request merged**